src/CC0-1.0.xml

```diff
@@ -6,7 +6,7 @@
          <crossRef>http://creativecommons.org/publicdomain/zero/1.0/legalcode</crossRef>
       </crossRefs>
       <titleText>
-         <p>Creative Commons CC0 1.0 Universal</p>
+         <p><optional>Creative Commons </optional>CC0 1.0 Universal</p>
```

Comment:
+1 for making “Creative Commons” optional.  Following [their `legalcode.txt`][1], we may want:

```xml
<titleText>
  <optional><p>Creative Commons<optional> Legal Code</optional></p><optional>
  <p>CC0 1.0 Universal</p>
</titleText>
```

Since [at least 2009-03-14][2], their HTML legal code has used `<h1>` and `<h2>` for those titles, but we don't have sized headers in our XML format (yet?).

[1]: https://creativecommons.org/publicdomain/zero/1.0/legalcode.txt
[2]: https://web.archive.org/web/20090314042016/https://creativecommons.org/publicdomain/zero/1.0/legalcode

Reply:
https://spdx.org/licenses/CC0-1.0.html has "Creative Commons CC0 1.0 Universal" all on one line though.